upgrading/topics/keycloak/changes-18_0_0.adoc

```diff
@@ -120,3 +120,6 @@ Components known to have breaking changes:
   
   - `noPadding` to `hasNoPadding`
 
+= Quarkus: Split metrics-enabled option into health-enabled and metrics-enabled
+
+The `metrics-enabled` option now only enables the metrics for Keycloak. To enable the readiness and liveness health endpoints, there's a new boolean option `health-enabled`. This allows more fine-grained usage of these options, e.g. enabling metrics but not enabling readiness/liveness probes for on-premise use cases. In order to keep the same behaviour as before when `metrics-enabled=true` was set, now also set `health-enabled=true` when building keycloak.
```

Comment:
```suggestion
The `metrics-enabled` option now only enables the metrics for Keycloak. To enable the readiness and liveness health endpoints, there's a new boolean option `health-enabled`. This allows more fine-grained usage of these options, e.g. enabling metrics but not enabling readiness/liveness probes for on-premise use cases. In order to keep the same behaviour as before when `metrics-enabled=true` was set, you need to additionally set `health-enabled=true` when building {project_name}.
```

Reply:
@vmuzikar could you then change your suggestion to use project_name also in the first sentence? ;)

Reply:
> @vmuzikar could you then change your suggestion to use project_name also in the first sentence? ;)

Done. Good catch. ;)

> btw: this file relies under the "keycloak" folder and there's a different one for rhsso, so I thought I didn't need to change it and looked up others that are also using static "Keycloak"..but whatever works for you.

Good point, but I think just to keep it consistent we should use the var. It is [already used](https://github.com/keycloak/keycloak-documentation/blob/3ddd5c2cfbe99b538806d4fc3d34929e9850930a/upgrading/topics/keycloak/changes-18_0_0.adoc#openid-connect-logout) in that file even.